src/main/java/com/artipie/asto/Storage.java

```diff
@@ -74,4 +75,12 @@
      * @return Bytes.
      */
     CompletableFuture<Flow.Publisher<Byte>> value(Key key);
+
+    /**
+     * Start a transaction with specified keys.
+     *
+     * @param keys The keys regarding which transaction is atomic
+     * @return transaction
+     */
+    CompletableFuture<TransactionStorage> transaction(List<Key> keys);
```

Comment:
@Sammers21 Please specify a more detailed JavaDoc. For example, does this method start a transaction for each ``Key``?